developments_build/sql/devdb_create_init.sql

```diff
@@ -74,6 +74,87 @@ IN PREVIOUS VERSION:
 */
 DROP TABLE IF EXISTS _INIT_devdb;
 WITH
+-- identify invalid dates
```

Comment:
should we use a function to check if dates are valid 
e.g. https://stackoverflow.com/questions/25374707/check-whether-string-is-a-date-postgresql
``sql
create or replace function is_date(s varchar) returns boolean as $$
begin
  perform s::date;
  return true;
exception when others then
  return false;
end;
$$ language plpgsql;
```

Reply:
I was looking for this! Thank you.

Reply:
But I wonder for dates like "05/10/2012". does that count as an invalid date? it would get turned into "2012-05-10", but are there special cases where it's supposed to be "2012-10-05"?

Reply:
It shouldn't happen because only europeans do DD/MM/YYYY, so it's safe to assume 2012-05-10 is correct?